test/crystal.jl

```diff
@@ -1,7 +1,6 @@
 using Test
-
-include("../src/CrystalShift.jl")
-#include("../src/crystal.jl")
+using CrystalShift
```

Comment:
In test suites, it's better practice to include the package you're testing, rather than including the code directly. If you have Revise.jl installed, this shouldn't slow you down.